File: library/src/main/java/com/sarang/timelinedetail2/AddCommentHolder.java
package com.sarang.timelinedetail2;

import android.view.LayoutInflater;
import android.view.ViewGroup;

import androidx.annotation.NonNull;

import com.sarang.timelinedetail2.databinding.ItemWriteCommentBinding;

public class AddCommentHolder extends DataBindingViewHolder<ItemWriteCommentBinding> {

    public static AddCommentHolder create(ViewGroup parent) {
        return new AddCommentHolder(ItemWriteCommentBinding.inflate(LayoutInflater.from(parent.getContext()), parent, false));
    }

    public AddCommentHolder(@NonNull ItemWriteCommentBinding binding) {
        super(binding);
    }

    public void setViewModel(TimeLineDetailViewModel timeLineDetailViewModel) {
        mBinding.setViewModel(timeLineDetailViewModel);
    }
}

File: library/src/main/java/com/sarang/feed_detail/holder/AddCommentHolder.kt
package com.sarang.feed_detail.holder

import android.view.LayoutInflater
import android.view.ViewGroup
import androidx.recyclerview.widget.RecyclerView
import com.sarang.feed_detail.databinding.ItemWriteCommentBinding

class AddCommentHolder(val binding: ItemWriteCommentBinding) : RecyclerView.ViewHolder(binding.root) {
    companion object {
        fun create(parent: ViewGroup): AddCommentHolder {
            return AddCommentHolder(
                ItemWriteCommentBinding.inflate(
                    LayoutInflater.from(parent.context),
                    parent,
                    false
                )
            )
        }
    }
}